pages/team.tsx

```diff
@@ -99,7 +96,35 @@ export default function Team() {
                   alt: `Headshot of ${name}`,
                 }}
               >
-                {description}
+                <p
+                  className="h-36 overflow-hidden line-clamp-5 mb-5 transition-[height] duration-300 ease-in-out whitespace-pre-line"
+                  style={{
+                    textOverflow: 'ellipsis',
+                  }}
+                >
+                  {description}
+                </p>
+                <Button
+                  fullWidth
+                  onClick={e => {
+                    const button = e.target as HTMLButtonElement;
+                    const paragraph = button.previousSibling as HTMLParagraphElement;
+                    paragraph.classList.toggle('expanded');
+                    const hasExpanded = paragraph.classList.contains('expanded');
+
+                    if (hasExpanded) {
+                      paragraph.style.height = `${paragraph.scrollHeight}px`;
+                      paragraph.classList.remove('h-36', 'line-clamp-5');
+                      button.innerText = 'Show Less';
+                    } else {
+                      paragraph.style.height = '';
+                      paragraph.classList.add('h-36', 'line-clamp-5');
+                      button.innerText = 'Show More';
+                    }
+                  }}
```

Comment:
I'm not really a fan of this for a few reasons:

1. I don't like using mutative JS at the same time as flow-based React. We should stick to one rendering methodology. In this case: React.
2. The goal it seems is to make the cards appear uniform in height; however, this solution is akin to sliding dirt under a couch. We've moved the layout shift to once the button is clicked.

Instead, I'd love if FlatCard's content had a capped height and scrollable content on overflow. Then there's never layout shift.